set.mm

```diff
@@ -52349,16 +52476,17 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
   $}
 
   ${
-    $d x y A $.  $d x y B $.  $d x y R $.
+    $d x y z A $.  $d x y z B $.  $d x y z R $.
     frc.1 $e |- B e. _V $.
     $( Property of well-founded relation (one direction of definition using
        class variables).  (Contributed by NM, 17-Feb-2004.)  (Revised by Mario
        Carneiro, 19-Nov-2014.) $)
     frc $p |- ( ( R Fr A /\ B C_ A /\ B =/= (/) ) ->
```

Comment:
It should be added why this proof became longer (because of less axioms!?).

Reply:
Quoting the PR description:

> Add rabeq0w -> use it in proofs of [frc](https://us.metamath.org/mpeuni/frc.html) , [frirr](https://us.metamath.org/mpeuni/frirr.html) to avoid ax-10, ax-11, ax-12 (credits and OLD proof not needed).

Change of [frc](https://us.metamath.org/mpeuni/frc.html) proof:
<pre>
1 frc.1        $e |- B e. _V
2 fri          $p |- ( ( ( B e. _V /\ R Fr A ) /\ ( B C_ A /\ B =/= (/) ) ) -> E. x e. B A. y e. B -. y R x )
3 1,2 mpanl1   $p |- ( ( R Fr A /\ ( B C_ A /\ B =/= (/) ) ) -> E. x e. B A. y e. B -. y R x )
4 3 3impb      $p |- ( ( R Fr A /\ B C_ A /\ B =/= (/) ) -> E. x e. B A. y e. B -. y R x )
5 rabeq0       $p |- ( { y e. B | y R x } = (/) <-> A. y e. B -. y R x )
6 5 rexbii     $p |- ( E. x e. B { y e. B | y R x } = (/) <-> E. x e. B A. y e. B -. y R x )
7 4,6 sylibr   $p |- ( ( R Fr A /\ B C_ A /\ B =/= (/) ) -> E. x e. B { y e. B | y R x } = (/) )
</pre>

<pre>
1 frc.1        $e |- B e. _V
2 fri          $p |- ( ( ( B e. _V /\ R Fr A ) /\ ( B C_ A /\ B =/= (/) ) ) -> E. x e. B A. z e. B -. z R x )
3 1,2 mpanl1   $p |- ( ( R Fr A /\ ( B C_ A /\ B =/= (/) ) ) -> E. x e. B A. z e. B -. z R x )
4 3 3impb      $p |- ( ( R Fr A /\ B C_ A /\ B =/= (/) ) -> E. x e. B A. z e. B -. z R x )
5 breq1        $p |- ( y = z -> ( y R x <-> z R x ) )
6 5 rabeq0w    $p |- ( { y e. B | y R x } = (/) <-> A. z e. B -. z R x )
7 6 rexbii     $p |- ( E. x e. B { y e. B | y R x } = (/) <-> E. x e. B A. z e. B -. z R x )
8 4,7 sylibr   $p |- ( ( R Fr A /\ B C_ A /\ B =/= (/) ) -> E. x e. B { y e. B | y R x } = (/) )
</pre>

The above edit is considered a "simple replacement", where the general proof idea stays the same, but a theorem is replaced with a weaker version using fewer axioms. It's the same kind of edit done in PR https://github.com/metamath/set.mm/pull/4203#discussion_r1766801376 where it was accepted that these changes don't need credits and OLD proofs.

Reply:
What about  "$( $j usage 'frc' avoids 'ax-10 , ..." ?

Reply:
> What about "$( $j usage 'frc' avoids 'ax-10 , ..." ?

Even those are typically not added https://github.com/metamath/set.mm/pull/4015 https://github.com/metamath/set.mm/pull/4176  https://github.com/metamath/set.mm/pull/3984, and for a good reason: most likely there are hundreds if not thousands of these edits, so it’s not practical to include any modification that goes beyond the replacement itself. For reference, reducing ax-13 usage required the proof edit of about 1500 theorems (done with the minimizer).

This PR contains only 3 of these simple replacements, so I have no problem including those tags here, I added them in a new commit [30c6fed](https://github.com/metamath/set.mm/pull/4256/commits/30c6fedde23193bfbe123593358a59b27b2b5d1f).

Reply:
> I am not talking about proofs whose axiom usage is implicitly reduced by reducing the axioms for theorems used in the proof, but about proofs which are explictly modified to reduce the axiom usage, as in this case.

Yes, I know. I am talking about those as well. The total amount of proofs that I edited with the minimizer in a span of 35 PRs have been about 1500. Then, the edit of those 1500 proofs caused ax-13 to be dropped by 31000 theorems considering indirect effects.